### ISSUE TITLE
Oops! (wrong spelling in the settings)

### ISSUE THREAD

**Issue by drewski-00h** [falzonv/discreet-launcher#69]
Under settings/help, "appearance" is spelled wrong.

**Comment by falzonv** [comment]
Hello,

Oh you are right, thank you for the remark I will fix it in the next release!
In French the translation is "Apparence" so probably I mixed both during a copy/paste :-/

Strangely enough, on the six times I wrote it in "strings_settings.xml" this was the only one spelled wrong, go figure...

```
<string name="title_appearance_settings">Appareance</string>
<string name="title_appearance_settings_general">General appearance</string>
<string name="title_appearance_settings_applications">Applications appearance</string>
```

Best regards.

**Comment by falzonv** [comment]
Hello,

I am pleased to inform you that this has been fixed in new release v4.1.0!
This release has been published a few minutes ago.

Here are the usual availability delays for each location:
- 4 to 8 days on F-Droid
- 0 to 2 days on Google Play
- Already available as direct APK on GitHub (https://github.com/falzonv/discreet-launcher/releases)

*Remarks:
I will leave this ticket opened until the release is effectively published on F-Droid.
You can use the export/import feature to switch from one version to another.*

Best regards.

**Comment by falzonv** [comment]
Since Thursday, release v4.1.0 is available on F-Droid so I will close this ticket.
Feel free to add comments if needed or wished.
